frontend/src/components/NotReadyProjectWarning.vue

```diff
@@ -0,0 +1,50 @@
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <v-tooltip top v-if="projectDetails.isNotReady">
```

Comment:
Why do we have a property `isNotReady` but not `isTerminating` and not `isInitial`. Shouldn't we also use `projectDetails.phase !== 'Ready'` for this case?